src/main/webapp/search-interview.html

```diff
@@ -106,25 +105,44 @@ <h5 class="text-center lead">Select the time you want from the dropdown on the d
         </form>
         <br>  
         <form>
+          <!-- TODO: Dynamically generate these with Sunday as the model. -->
+          <!-- TODO: Make the week Sunday - Saturday. -->
           <div class="form-row">
             <div class="col-3 date-label">
-              <label>Saturday 7/4</label>
+              <label>Sunday 7/5</label>
             </div>
             <div class="col-5">
-              <select class="form-control">
+              <select class="form-control" id="7/5/2020-options">
                 <option>9:30 AM - 10:30 AM</option>
                 <option>6:30 PM - 7:30 PM</option>
               </select>
             </div>
             <div class="col-4">
-              <!-- TODO: change onclick to addEventListener inside loadInterviews() so it can pass in params. -->
-              <button type="button" class="btn btn-primary mb-2" onclick="selectInterview()">Select</button>
+              <button type="button" class="btn btn-primary mb-2" 
+              onclick="showInterviewers(this)" id="7/5/2020">Select</button>
             </div>
           </div>
         </form>
         <br>  
       </section>
+      
```

Comment:
Is this space necessary?

Reply:
l guess not but l like how it separates the modal from the other stuff.